apiserver/plane/bgtasks/work_item_link_task.py

```diff
@@ -57,17 +46,18 @@ def crawl_work_item_link_title_and_favicon(url: str) -> Dict[str, Any]:
             "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36"  # noqa: E501
         }
 
-        # Fetch the main page
-        response = requests.get(url, headers=headers, timeout=2)
+        soup = None
```

Comment:
@NarayanBavisetti Since we will be passing `None` to the favicon function for `soup`, we need to handle that case.